ci/cleanup/aws.py

```diff
@@ -125,6 +123,28 @@ def clean_up_iam() -> None:
             print(f"Skipping role {role_name}")
 
 
+def get_testdrive_roles(client: Any) -> list[Any]:
+    roles = []
+    marker = None
+
+    while True:
```

Comment:
I guess not too much danger of an endless loop, timeout is 10 min anyway

Reply:
Yeah. There is a break statement, which is reached when `is_truncated` is `False`.